chat_server/db/db.env

```diff
@@ -0,0 +1,5 @@
+PGHOST=db
+PGUSER=docker
+PGPASSWORD=docker
+PGDATABASE=portail_message
+PGPORT=5432
```

Comment:
Move to root folder

Reply:
This is yet only used by the container I use in the chat_server... Maibe we should update the hole config yet